CMakeLists.txt

```diff
@@ -132,12 +132,17 @@ if (("${SEQAN_BUILD_SYSTEM}" STREQUAL "SEQAN_RELEASE") OR
     add_subdirectory (demos)
 endif ()
 
-message (STATUS "Configuring apps")
-add_subdirectory (apps)
+if (NOT "${SEQAN_BUILD_SYSTEM}" STREQUAL "SEQAN_RELEASE_LIBRARY")
+    message (STATUS "Configuring apps")
+    add_subdirectory (apps)
+endif ()
 message (STATUS "Configuring dox")
 add_subdirectory (dox)
-message (STATUS "Configuring manual")
-add_subdirectory (manual)
+if ((NOT "${SEQAN_BUILD_SYSTEM}" STREQUAL "SEQAN_RELEASE_LIBRARY") AND
+    (NOT "${SEQAN_BUILD_SYSTEM}" STREQUAL "SEQAN_RELEASE_APPS"))
+    message (STATUS "Configuring manual")
```

Comment:
This is not yet fully correct, as we don't want the manual either when we have APP:*

I propose
```cmake
if ("${SEQAN_BUILD_SYSTEM}" STREQUAL "DEVELOP")
    message (STATUS "Configuring manual")
    add_subdirectory (manual)
endif ()
```
and also for dox:
```cmake
if (("${SEQAN_BUILD_SYSTEM}" STREQUAL "DEVELOP") OR
    ("${SEQAN_BUILD_SYSTEM}" STREQUAL "SEQAN_RELEASE_LIBRARY"))
    message (STATUS "Configuring dox")
    add_subdirectory (dox)
endif ()
```